### PR TITLE
Add RockyLinux 8 support

### PR DESCRIPTION
**Pull request by vbalys**
Add support for RockyLinux 8 and drop support for Debian/Ubuntu. 

Test with MySQL 5.7 community server. 

Bump Ruby to v3, change some other things to satisfy Rubocop.

### CODE REVIEW

**Review by aliulis — APPROVED** [review]

File: libraries/base_service.rb [hunk 1]
```diff
@@ -93,37 +94,19 @@ def create_user
...
+        end
 
-          remote_file repo_deb do
-            source url
-            notifies :install, "dpkg_package[#{basename}]", :immediate
-            not_if { ::File.exist?(repo_deb) }
-          end
+        package 'proxysql' do
+          action :nothing
         end
```
Comment (vytstank) [inline libraries/base_service.rb:104]: not intuitive from resource name - do you need package install here or this is used to flush package cache?

**Review by vytstank — APPROVED** [review]
otherwise looks ok

File: libraries/base_service.rb (hunk 1 above)
Comment (vbalys) [reply libraries/base_service.rb:104]: Only to flush package cache. Will check - maybe this flush can be relocated to where actual install takes place.

**Review by dmitry-g — APPROVED** [review]

File: libraries/base_service.rb (hunk 1 above)
Comment (vbalys) [reply libraries/base_service.rb:104]: Did exactly that
https://github.com/vinted/chef-proxysql/pull/55/files#diff-10a18e913f01b118de81cd2d7248a5189c6d1b5fbb2949e40ab381cadd92bda3R313

**Pull request merged**